### ISSUE TITLE
Feature Request - add second WiFi configuration

### ISSUE THREAD

**Issue by wib100** [tbnobody/OpenDTU#66]
Hi,
would be great if there would be an option to add a second WiFi/Hotspot to the network configuration.
This would be very helpful to pre-configure an Arduino.

Thanks,
Willi

**Comment by Dom1n1c** [comment]
I would appreciate it. Some more or less known projects and ESP based devices such as: ESPeasy, Tasmota (depending on the release), Shelly also have the possibility of adding a secondary or backup WiFi.

**Comment by ortegafernando** [comment]
> I would appreciate it. Some more or less known projects and ESP based devices such as: ESPeasy, Tasmota (depending on the release), Shelly also have the possibility of adding a secondary or backup WiFi.

I agree you, Tasmota for example has also a secondary or backup Wifi. I have also found it usefull when you want to change your main wifi password, you will never lost connection with opendtu if it has two wifi "configs".
